.github/workflows/release_community_cromwell.yml

```diff
@@ -0,0 +1,99 @@
+name: Release Community Cromwell
+
+on:
+  workflow_dispatch: 
+
+jobs:
+  cromwell-release:
+    name: Release Community Cromwell
+    runs-on: ubuntu-latest
+    steps:
+    - uses: sbt/setup-sbt@v1
+
+    # `DSDEJENKINS_PASSWORD` auto syncs from vault with https://github.com/broadinstitute/terraform-ap-deployments/pull/614
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: dsdejenkins
+        password: ${{ secrets.DSDEJENKINS_PASSWORD }}
+
+    - name: Find Cromwell release number
+      run: |
+        set -e
+        previous_version=$(curl -X GET https://api.github.com/repos/broadinstitute/cromwell/releases/latest | jq .tag_name | xargs)
+        if ! [[ "${previous_version}" =~ ^[0-9][0-9]+$ ]]; then
+          exit 1
+        fi
+        echo "RELEASE_VERSION=$((previous_version + 1))" >> $GITHUB_ENV
```

Comment:
Are cromwell versions always incremented by 1? Would it make sense to have a manual override using workflow inputs in case this changes?

Reply:
In the past they have always been incremented by 1, and there are a few fiddly things in here that would go wrong if we started using arbitrary versions.  We'll need to address this as part of switching to semantic versioning, which we should do, but that's a conversation for a different PR - are you okay waiting until then to do custom version support?

Reply:
This logic does replicate the old release script.